IPA-End-to-End.md

```diff
@@ -172,15 +177,16 @@ Random noise is added to each breakdown, using ε and the _sensitivity_ to infor
 
 ### Differential Privacy Budget Management
 
-The previous section focuses on applying differential privacy to individual queries. However, we need to further design a system such that over an _epoch_, the amount of information released about people is bounded. Specifically, we propose that for a given _epoch_ and _website/app_ (represented by a single _report collector_), individuals (represented by _match keys_) can have a bounded impact on the results, as measured by ε differential privacy.
+The previous section focuses on applying differential privacy to individual queries. However, we need to further design a system such that over an _epoch_, the amount of information released about people is bounded. Specifically, we propose that for a given _epoch_ and _website/app_ (represented by one or more _report collectors_), individuals (represented by _match keys_) can have a bounded impact on the results, as measured by ε differential privacy.
+
+In our current approach, we achieve this by providing each _website/app_ with a budget, ε. That _website/app_ is then able to allocate that budget across their _report collectors_ such that each _report collector_ has budget ε<sub>r</sub> (and the sum of all ε<sub>r</sub> = ε. _Websites/apps_ will need to commit to a specific budget allocation, and will not be able to change that commitment until the next epoch. The mechanism for managing this commitment is an open problem.
 
- In our current approach, we achieve this by providing each _report collector_ with a budget, ε for the given _website/app_ they are querying on behalf of. When _report collectors_ run queries they will specify how much budget to use, ε<sub>i</sub>, which will be deducted from their remaining budget. For example, given an _epoch_ limit of ε, a _report collector_ could perform 10 queries, each with global DP applied at ε/10, or a more complicated set of queries such as three with ε/5 and four with ε/10. (Note that smaller ε<sub>i</sub> result in _more_ noise and, thus, _more_ privacy.)
+When _report collectors_ run queries they will specify how much budget to use, ε<sub>i</sub>, which will be deducted from their remaining budget. For example, given an _epoch_ limit of ε, a _report collector_ could perform 10 queries, each with global DP applied at ε/10, or a more complicated set of queries such as three with ε/5 and four with ε/10. (Note that smaller ε<sub>i</sub> result in _more_ noise and, thus, _more_ privacy.)
```

Comment:
Does this paragraph need to use $\epsilon_r$ ?

Reply:
it's probably clearer, yes. I'll update accordingly